test/ng/directive/ngModelOptionsSpec.js

```diff
@@ -459,11 +459,12 @@ describe('ngModelOptions', function() {
           $rootScope.$watch(watchSpy);
 
           helper.changeInputValueTo('a');
+          expect(function() {$browser.defer.flush();}).toThrowError('No deferred tasks to be flushed');
```

Comment:
This doesn't work correctly. It still flushes the task and triggers a digest. I haven't found out why exactly it doesn't work, but I suspect it's because of a mismatch between the real browser and the mock browser.